### ISSUE TITLE
While executing the code an error is shown

### ISSUE THREAD

**Issue by goscha01** [scrimba/community#551]
After executing the code which is written by the lecturer, a message Missing cw1 is rendered (s.attachemnt)
![scrimbaBug](https://user-images.githubusercontent.com/43910293/95521666-17912800-0998-11eb-8b65-5831732f18e1.png)

**Comment by wizozz** [comment]
I just got this same issue while doing module 7, simplify Constructor function 1. And it is not even about multiple tabs. It was persistent even after closing all the other tabs. 

I spent around an hour stewing on this feeling stupid *lol*
![screenshot-scrimba com-2022 05 20-08_24_05](https://user-images.githubusercontent.com/19334110/169537610-ed7c1ea8-d5ed-4e45-90a9-87d335751ec8.png)

**Comment by ms1241721** [comment]
Anyone , i am also facing the same issue

**Comment by shaayesteh** [comment]
Has anyone solved this problem? I get this error and have no idea what the problem is.

**Comment by dufmel** [comment]
Getting the same issues. I've closed all tabs and still getting the problem.

**Comment by malpal420** [comment]
i am getting missing cw2 error

**Comment by krishnadiv** [comment]
i am getting Missing cw0 error

**Comment by smajkut** [comment]
I have the same problem, "Missing cw0", when I'm trying compile the code

**Comment by Kiefna** [comment]
Same Missing CW1

**Comment by blockdevjpm** [comment]
Yeah I got the same problem! Hope you will come with a quick fix!

**Comment by malpal420** [comment]
I restarted my browser and the problem seemed to go away! try and see if that worksSent from my iPhoneOn Feb 3, 2023, at 11:57 AM, blockdevjpm ***@***.***> wrote:﻿
Yeah I got the same problem! Hope you will come with a quick fix!

**Comment by Kiefna** [comment]
I can confirm this is happening regardless of whether I have multiple tabs open.

**Comment by Arthavruksha** [comment]
Facing the same issue !

**Comment by Adaennnn** [comment]
Has anyone been able to fix this? 

I'm getting the "Missing cw0" on the preview.

**Comment by ShujaShah** [comment]
Same issue, i am facing the error missing cw6

**Comment by intondiscott** [comment]
I believe this has to do with popups in the browser. Allow popups from Scrimba site you should be good to go.

**Comment by shivampandey777** [comment]
I am face the same issue on preview cw0 Module 10.

**Comment by zeeshanchana95** [comment]
I am also facing the same issue of **missing cw()** in scrimba mini browser.

**Comment by zeeshanchana95** [comment]
> I am also facing the same issue of **missing cw()** in the scrimba mini browser.

I don't know why that problem occurs but after refreshing the tap, problem is fixed. Just thinking why actually that error occurs.

**Comment by Cheekay** [comment]
Hey there,

Looks like the last comment here was over 3 weeks ago. Is anyone else still experiencing this issue? 
I was just going through the CSS grid tutorial and encountered the error (Missing cw0)

<img width="982" alt="Screen Shot 2023-10-20 at 5 00 14 PM" src="https://github.com/scrimba/community/assets/7269504/706abf1d-add2-4034-93f1-beecec06e940">

**Comment by zeeshanchana95** [comment]
> Hey there,
> 
> Looks like the last comment here was over 3 weeks ago. Is anyone else still experiencing this issue? I was just going through the CSS grid tutorial and encountered the error (Missing cw0)
> 
> <img alt="Screen Shot 2023-10-20 at 5 00 14 PM" width="982" src="https://user-images.githubusercontent.com/7269504/277051114-706abf1d-add2-4034-93f1-beecec06e940.png">

Hi @Cheekay , 

I don't know exactly the reason for this error but I fixed it by refreshing the browser tab. There are couple of other solutions and references given by students above, so you can check that out, it might be helpful.

Thanks.

**Comment by TheArgentinian** [comment]
We're paying for a broken service. Ridiculous!

**Comment by chrislizardc** [comment]
I am still getting missing cw0 ? Any fixes yet?

**Comment by Alyyy-Ayman** [comment]
iam facing the missing cw0 issue too , its just out of no where , every singl time i refresh my mini browser it says " missing cw0 " .. please help its stopping me from checking my code in the mini browser !

![IMG-20240224-WA0000](https://github.com/scrimba/community/assets/160285443/cc59bcb3-0b82-4d33-bef1-21f7c1bbe479)
